data-collection/src/main/java/refactoringml/db/CommitMetaData.java

```diff
@@ -0,0 +1,63 @@
+package refactoringml.db;
+
+import org.eclipse.jgit.revwalk.RevCommit;
+import refactoringml.ProcessMetric;
+import refactoringml.util.JGitUtils;
+
+import javax.persistence.*;
+import java.util.Calendar;
+
+@Entity
+@Table(name = "commit_meta-data")
+public class CommitMetaData {
```

Comment:
You also need to add this class to the `HibernateConfiguration` class. See it works for the other classes: https://github.com/mauricioaniche/predicting-refactoring-ml/blob/master/data-collection/src/main/java/refactoringml/db/HibernateConfig.java#L59